maestro-cli/src/main/java/maestro/cli/util/ErrorReporter.kt

```diff
@@ -0,0 +1,43 @@
+package maestro.cli.util
+
+import maestro.cli.api.ApiClient
+import picocli.CommandLine
+import java.security.MessageDigest
+import java.util.concurrent.Executors
+import java.util.concurrent.TimeUnit
+import kotlin.Exception
+
+object ErrorReporter {
+
+    private const val BASE_API_URL = "https://api.mobile.dev"
+    private val executor = Executors.newCachedThreadPool {
+        Executors.defaultThreadFactory().newThread(it).apply { isDaemon = true }
+    }
+
+    fun report(exception: Exception, parseResult: CommandLine.ParseResult) {
+        val args = parseResult.expandedArgs()
+        val scrubbedArgs = args.mapIndexed { idx, arg ->
+            if (idx > 0 && args[idx - 1] in listOf("-e", "--env")) {
+                val (key, value) = arg.split("=", limit = 1)
+                key + "=" + hashString(value)
+            } else arg
+        }
+
+        executor.submit {
+            ApiClient(BASE_API_URL).sendErrorReport(
+                exception,
+                scrubbedArgs.joinToString(" ")
+            )
+        }.get(1, TimeUnit.SECONDS)
```

Comment:
What happens if this request fails? Will there be an error printed to users or will it be swallowed and hidden from them?

Reply:
I believe all of this reporting stuff should be invisible to the user, as this is not the primary concern of the user when running maestro. Fix here: https://github.com/mobile-dev-inc/maestro/pull/584/commits/979370c1cd94d25720f5fc88ee2b4b11db1f18d9

Reply:
Excellent, I completely agree 🙌